dev/com.ibm.ws.security.openidconnect.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -298,3 +298,8 @@ pkceCodeChallengeMethod.S256=S256
 
 tokenRequestOriginHeader=Token request origin header
 tokenRequestOriginHeader.desc=Specifies the value to use in the Origin HTTP header that is included in the HTTP POST request to the token endpoint of the OpenID Connect provider. If not specified, an Origin HTTP header is not included in the request.
+
+tokenOrderToFetchCallerClaims=Specify one or more tokens to fetch caller claims 
+tokenOrderToFetchCallerClaims.desc=Specifies the order of the token/s to fetch the caller name and group claim values. If the claim does not exist in the first token, then the OpenID Connect client will continue the search with other tokens in the list
```

Comment:
- Change "the order of the token/s" to "token order"

Info: Current style guidelines ask us to use plural ("tokens" instead of "token/s"). With the suggested change you don't need to specify plural.

- Change "will continue" to "continues"
- Add a period to the end of the last sentence

Reply:
updated the description per  your comments , its done here - https://github.com/OpenLiberty/open-liberty/pull/26719/commits/ee5dcddd3d79f39cbbc77d62f5a9a3f4e80f6fd8